### PR TITLE
Replaced dummy images

### PR DESCRIPTION
**Pull request by HakimSkaik**
Replaced all images to CUSEC conferences, please have a look, PS:  the images at the "what is CUSEC?" seem to be a bit crammed, we can handle that in another pr issue

### CODE REVIEW

File: src/components/Image/Image.scss [hunk 1]
```diff
@@ -1,7 +1,8 @@
 .Image {
-  width: 100%;
-  height: auto;
-  border: solid white 0px;
+  width: 150%;
+  height: 100%;
```
Comment (danielle-dsouza) [inline src/components/Image/Image.scss:3]:
```suggestion
  width: 100%;
  object-fit: cover;
```

Comment (danielle-dsouza) [reply src/components/Image/Image.scss:3]: This should fix the cramped issue.

**Review by danielle-dsouza — CHANGES_REQUESTED** [review]
Looks really good! Thanks for finding the images!

File: src/components/Image/Image.scss [hunk 2]
```diff
@@ -1,6 +1,6 @@
 .Image {
   width: 150%;
```
Comment (danielle-dsouza) [inline src/components/Image/Image.scss:2]: Is there a reason why we have 150%?

Comment (HakimSkaik) [reply src/components/Image/Image.scss:2]: it decreases the size of the other images significantly, making them look tiny next to the text

Comment (HakimSkaik) [reply src/components/Image/Image.scss:2]: 150%: 
image
100%:
image

**Review by danielle-dsouza — APPROVED** [review]

File: src/components/Image/Image.scss (hunk 2 above)
Comment (danielle-dsouza) [reply src/components/Image/Image.scss:2]: Thanks!

**Pull request merged**